README.md

```diff
@@ -263,12 +265,27 @@ This is the default directory structure we use for this project:
 
 ```
 leto-modelizer-plugin-core
+├ cypress              ⇨ Contains all the cypress related files
+│ ├ e2e                ⇨ Contains all the e2e tests
+│ └ support            ⇨ Contains all support files for e2e tests
+│   └ step_definitions ⇨ Contains all step definitions for e2e tests
+├ demo                 ⇨ Contains a vue demo application
+├ dist                 ⇨ Contains the built application
 ├ docs                 ⇨ Contains all files generated by esdoc
+├ public               ⇨ Contains all public files, like favicon
 ├ reports              ⇨ Contains all the report files for sonar
 ├ guides               ⇨ Contains all the guides
+├ └  docs              ⇨ Contains all plugin-core documentation
 │ └  migrations        ⇨ Contains all migration guides
+│ └  svg               ⇨ Contains the guide to build a component model
 ├ src                  ⇨ Contains all files for modeling tools
-│ └  models            ⇨ Contains all the models
+│ ├ assets             ⇨ Contains all the assets
+│ ├ draw               ⇨ Contains all files related to the Class that draws a component in a graphical representation
```

Comment:
Contains all files related to the Class that draws components in a graphical representation

Reply:
done